vmware_dvs/agent/firewalls/dvs_securitygroup_rpc.py

```diff
@@ -13,6 +13,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
+from threading import Timer
```

Comment:
Please, swap this 2 imports:
`from oslo_log import log as logging` should be placed before  `import Timer`